cost-analyzer/values.yaml

```diff
@@ -418,6 +418,16 @@ kubecostModel:
   # For deploying kubecost in a cluster that does not self-monitor
   etlReadOnlyMode: false
 
+  ## Feature to view your out-of-cluster costs and their k8s utilization
+  ## Ref: https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/cloud-costs-explorer
+  cloudCost:
+    enabled: false
+    labelList:
+      isIncludeList: false
```

Comment:
This should be a capital `I`
--set kubecostModel.cloudCost.labelList.IsIncludeList=true
no?

unless we want to change the template? because camelCase would be lowercase i?

Reply:
@jessegoodier Great catch, thank you!

I'd prefer for it to be `isIncludeList` to be consistent with how all other configs are named. However I think it's best we don't adjust the template to do so. Don't want to cause any breaking changes for existing users already configuring `IsIncludeList`.